src/providers/address/address.spec.ts

```diff
@@ -57,135 +61,78 @@ describe('AddressProvider', () => {
-      let BCHTestnetAddress = 'qqr99gt5qdk4qyaxxvzeapgjuxkg6x9ueue95fakj7';
-      let result = addressProvider.validateAddress(BCHTestnetAddress);
-      expect(result).toEqual({
-        address: 'qqr99gt5qdk4qyaxxvzeapgjuxkg6x9ueue95fakj7',
-        isValid: true,
-        network: 'testnet',
-        coin: 'bch',
-        translation: {
-          origCoin: 'bch',
-          origAddress: 'qqr99gt5qdk4qyaxxvzeapgjuxkg6x9ueue95fakj7',
-          resultCoin: 'btc',
-          resultAddress: 'mg6PLV5yyUS6Gy55fJ7f994dQ7RpfJNYC9'
-        }
-      });
-    });
 
-    it('should return invalid values if we send an invalid address', () => {
-      let address = 'invalidAddress';
-      let result = addressProvider.validateAddress(address);
-      expect(result).toEqual({
-        address: 'invalidAddress',
-        isValid: false,
-        network: 'livenet',
-        coin: null,
-        translation: undefined
-      });
+    it('should return null if we send an invalid address', () => {
+      const address = 'invalidAddress';
+      const result = addressProvider.getCoin(address);
+      expect(result).toBeNull();
     });
   });
 
   describe('getNetwork', () => {
-    it('should return correct network for BTC livenet address', () => {
-      let BTCLivenetAddress = '15qT4RJTjs7GSTReEmgXr4LbMjTVQ51LZA';
-      let result = addressProvider.getNetwork(BTCLivenetAddress);
-      expect(result).toEqual('livenet');
+    const BTCLivenetAddresses = [
+      '15qT4RJTjs7GSTReEmgXr4LbMjTVQ51LZA',
+      'bitcoin:1CVuVALD6Zo7ms24n3iUXv162kvUzsHr69',
+      'bitcoin:1CVuVALD6Zo7ms24n3iUXv162kvUzsHr69?amount=0.00090000'
+    ];
 
-      BTCLivenetAddress = 'bitcoin:1CVuVALD6Zo7ms24n3iUXv162kvUzsHr69';
-      result = addressProvider.getNetwork(BTCLivenetAddress);
-      expect(result).toEqual('livenet');
-    });
+    const BTCTestnetAddresses = [
+      'mscoRRUxbicZdUms3EqDr9jwtCmbbPgfcY',
+      'bitcoin:n3LHh1WTFSpSVKXNFQo4U5eLAqowCadFHY',
+      'bitcoin:n3LHh1WTFSpSVKXNFQo4U5eLAqowCadFHY?amount=0.00090000'
+    ];
 
-    it('should return correct network for BTC testnet address', () => {
-      let BTCTestnetAddress = 'mscoRRUxbicZdUms3EqDr9jwtCmbbPgfcY';
-      let result = addressProvider.getNetwork(BTCTestnetAddress);
-      expect(result).toEqual('testnet');
+    const BCHLivenetAddresses = [
+      'CHUwf57Maceqk5jhMX6NJkuLEbHEfqqgwx', // BCH Livenet Bitpay Style
+      'qq9jk8jskjsmgqnzygwjsghp3knautm2dcnc5e4k7n',
+      'bitcoincash:qz8ds306px5n65gffn8u69vvnksfw6huwyjczrvkh3',
+      'BITCOINCASH:qz8ds306px5n65gffn8u69vvnksfw6huwyjczrvkh3',
+      'bitcoincash:qz8ds306px5n65gffn8u69vvnksfw6huwyjczrvkh3?amount=0.00090000'
+    ];
 
-      BTCTestnetAddress = 'bitcoin:n3LHh1WTFSpSVKXNFQo4U5eLAqowCadFHY';
-      result = addressProvider.getNetwork(BTCTestnetAddress);
-      expect(result).toEqual('testnet');
-    });
+    const BCHTestnetAddresses = [
+      // 'mg6PLV5yyUS6Gy55fJ7f994dQ7RpfJNYC9', // TODO: BCH Testnet Bitpay Style
```

Comment:
add: 
`BCHTEST: mg6PLV5yyUS6Gy55fJ7f994dQ7RpfJNYC9`

Reply:
we have to update BWS with bitcore-lib-cash version 0.18.1 to get correct values for BCHTEST prefix